tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+<2> The `Normative references` section contains references to related standards. The compiler needs the `[bibliography]` tag, so that it renders the reference correctly.
+<3> An alternative way to assign a section identifier is use the `heading` tag. 
+<4> Here, the appendix is called “Annex”, and needs the `appendix` tag to define what kind of section it is. You can specify if a section is normative or informative by adding the `obligation` identifier after the section identifier. 
+
+[[exercise-2-2]]
+include::exercises.adoc[tag=exercise-2-2]
+
+We’ve built the basic structure of a Metanorma document. Let’s add some content to our document in the next lesson.
+>> Start next lesson 
+
+=== Blocks
+
+Information that forms a logical segment, such as a paragraph or a list, is called a block.
+A block starts and ends with a https://asciidoctor.org/docs/asciidoc-writers-guide/#delimited-blocks[delimiter] which is a sequence of characters. The delimiters tell the compiler that the following input belongs together. 
+
+.Examples for different blocks
+[source, AsciiDoc]
+------
+I’m a paragraph and I don’t need a block delimiter. 
+<1>
+. I’m a list item and also do not need a delimiter
+. I’m the second list item
+. I'm list item number three
+
+//// <2>
+I’m a comment.
+////
+
+[source, python] <3>
+----
+print("Hello World!")
+----
+------
+
+<1> Paragraphs and lists are types of blocks that do not need any delimiters. They are separated by a blank line in between blocks.
+<2> Comments use four slashes as a delimiter `////`. No other block uses slashes to begin and end a block.
+<3> All other types of blocks rely on delimiters, such as four dashes `----` or four equal signs `====`. This block contains a code sample written in python.
+
+
+Apart from the blocks shown in the example, there are many types of blocks in AsciiDoc, such as:
+* Paragraphs
+* Lists
+* Tables
+* Images
+* Admonitions (Note, Caution, Warning, etc.)
+* Code samples
+* Term definitions
+* Comments
+* Reviewer Notes
+
+Let’s look at lists in the next lesson. 
+>> Next lesson
+
+==== Lists
+
+Metanorma AsciiDoc supports three types of lists:
+
+* Unordered lists
+* Ordered lists
+* Definition lists
+
+*Unordered lists*
+
+Unordered lists are bulleted and can be nested. They look like this:
+
+[source, AsciiDoc]
+----
+The main changes compared to the previous edition are:
+
+* updated normative references;
+* deletions:
+** deletion of 4.3. [nested list item]
+** deletion of 4.4. [nested list item]
+----
+
+*Ordered lists* 
+
+Ordered lists are invoked by beginning the line with a dot `.`. The list items are numbered automatically. The list style depends on the stylesheet of your organization. However, if you want to tweak how your list looks like, you can use an attribute before the list starts, for example:
```

Comment:
Custom numbering is not supported in all flavours, and I don't want to support it in all flavours. I'd prefer this to be removed.